File: src/QuickApp/ViewModels/Custom/ReportVm.cs
using FSW_TFS.Web.ViewModels.Helpers;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FSW_TFS.Web.ViewModels.Custom
{
    public class ReportVm
    {
        public int Id { get; set; }
        public string ShortDescription { get; set; }
        public SelectItem Priority { get; set; }
        public string FullDescription { get; set; }
        public string ClientId { get; set; }
        public int Status { get; set; }
    }
}

File: src/QuickApp/ClientApp/src/app/models/viewModels/reportVm.ts
import { SelectItem } from '../../../../node_modules/primeng/primeng';

export interface IReport {
    id: number;
    shortDescription: string;
    priority: SelectItem;
    fullDescription: Date;
    clientId: string;
    status: number;
}

export class Report implements IReport {
    id: number;
    shortDescription: string;
    priority: SelectItem;
    fullDescription: Date;
    clientId: string;
    status: number;
}
